src/sentry/models/userrole.py

```diff
@@ -0,0 +1,34 @@
+from typing import FrozenSet
+
+from django.db import models
+
+from sentry.db.models import ArrayField, Model, sane_repr
+
+
+class UserRole(Model):
```

Comment:
Could we add a `date_added` field here? One option is to just inherit from `DefaultFieldsModel` and you'll get `date_added/updated` for free